packages/tasit-action/src/NFT.test.js

```diff
@@ -262,28 +250,24 @@ describe("TasitAction.NFT", () => {
       expect(contractErrorFakeFn.called).to.be.true;
 
       senderBalance = await fullNFT.balanceOf(sender);
-      expect(senderBalance.toNumber()).to.equal(1);
-
       receiverBalance = await fullNFT.balanceOf(receiver);
-      expect(receiverBalance.toNumber()).to.equal(0);
+
+      expect([
```

Comment:
Probably overkill, but we could make a tiny function called `confirmReceipt(sender, receiver)` that gets the balances and does the expectations

Reply:
I've just created a function `confirmBalances(addresses, balances)`

Reply:
Great. Resolving this.